improver_tests/precipitation_type/convection/test_DiagnoseConvectivePrecipitation.py

```diff
@@ -460,44 +378,6 @@ def test_basic(self):
         """Test that the sum of differences between adjacent grid squares,
```

Comment:
This test and the one below are the same, except one has a scalar realization coord and one has a dim realization coord. So only kept the 2D versions of the tests